contrib/root/usr/share/scripts/patroni/post_init.sh

```diff
@@ -2,8 +2,14 @@
 set -Eeu
 
 if [[ (! -z "$APP_USER") &&  (! -z "$APP_PASSWORD") && (! -z "$APP_DATABASE")]]; then
-  echo "Creating user ${APP_USER}"
-  psql "$1" -w -c "create user \"${APP_USER}\" WITH LOGIN ENCRYPTED PASSWORD '${APP_PASSWORD}'"
+
+  if [[ "$APP_USER" = "postgres" ]]; then
```

Comment:
This is a little different behaviour than most postgres setups.  Most have an `POSTGRES_ADMIN_PASSWORD` as a separate parameter to change the master user password.

Reply:
Agree to separate admin and regular user (the user the application uses).

Reply:
I like that. I'll make the change.

I do still have on issue when trying to run aqua, since it uses the default user by default, it is impossible to run this image without specifying a user. 

Perhaps a check for ` APP_USER != "postgres" ` before creating the user is necessary, otherwise we print out a statement saying `APP_USER is default user - consider changing`

Reply:
Postgres always has the default master account `postgres`.  A default installation will not assign a password, but will only allow localhost traffic to use the account.  When you assign the password during initialization of the db it enables network access to the `postgres` account.
If you want aqua to use the `postgres` account, you probably just need to set the password for the account.
If you want aqua to use a different account, you'll probably need to setup that user with the correct permissions.

Reply:
Understood, but this script attempts to create the user `postgres` and subsequently fails

Reply:
Then most likely it should fail in a nice way, as the `postgres` user will always exist and it shouldn't be attempted to be created.

Reply:
yes exactly. LIke this script should not attempt to create a postgres user and instead just use it.

Reply:
Look into the environment variables `PATRONI_SUPERUSER_USERNAME` and `PATRONI_SUPERUSER_PASSWORD`. IIRC they're generally associated with the postgres superuser account. If those environment variables are specified (passed in via secret mapping on dc or similar) Patroni will use that for account management of `postgres` user and its associated password.

Understanding this correctly, we probably should not be doing this new if-else logic for handling/updating the postgres superuser account as there's ENV variables that deal with it already.